frontend/util/assets.ts

```diff
@@ -0,0 +1,9 @@
+const icon = import.meta.globEager(`@/assets/icon/*.png`);
+export const getIconPath = (name: string): string => {
+  for (const path in icon) {
+    if (path.includes(name)) {
```

Comment:
It might be not that tough to fuzzy match a module name. Do you think we can introduce a stronger way to find the icon?